utils/atomsPropTableData.ts

```diff
@@ -0,0 +1,56 @@
+type AtomsProp = {
+    name: string;
+    required: boolean;
+    description: string
+}
+
+export const googleCalendarAtomPropsData: AtomsProp[] = [
+    {name: 'className', required: false, description: 'To pass in custom classnames from outside for styling the atom'},
+    {name: 'label', required: false, description: 'The label for the connect button'}, 
+    {name: 'alreadyConnectedLabel', required: false, description: 'The label for the already connected button'}, 
+    {name: 'onCheckError', required: false, description: 'A callback function to handle errors when checking the connection status'}
+]
+
+export const availabilitySettingsAtomPropsData: AtomsProp[] = [
+    {name: 'id', required: false, description: `The id of the schedule which fetches a user's availability`},
+    {name: 'labels', required: false, description: 'Helpful if you wanna pass in your own lables for i18n'},
+    {name: 'customClassNames', required: false, description: 'To pass in custom classnames from outside for styling the atom'}, 
```

Comment:
Availability and booker "Props" section has customClassNames, but in "Styling" section customClassNames is missing, so I think customClassNames should also appear in availability and booker classname table.

Reply:
I'll write some info around this since Google calendar atom only accepts custom classnames via the `classNames` while availability and booker accept styling via the `customClassNames` prop